Makefile

```diff
@@ -124,3 +124,10 @@ fmt-code: ## Run go fmt against code.
 
 vet: ## Run go vet against code.
 	go vet ./...
+
+generate-api:
+	hack/verify-codegen.sh
```

Comment:
the script this is running doesnt exist, same for install-tools below.. should these be cleaned up as well?